pkg/kubectl/cmd/resource/get.go

```diff
@@ -666,6 +707,10 @@ func addOpenAPIPrintColumnFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(useOpenAPIPrintColumnFlagLabel, true, "If true, use x-kubernetes-print-column metadata (if present) from the OpenAPI schema for displaying a resource.")
 }
 
+func addServerPrintColumnFlags(cmd *cobra.Command) {
+	cmd.Flags().Bool(useServerPrintColumns, true, "If true, have the server return the appropriate table output. Supports extension APIs and CRD. Experimental.")
```

Comment:
IIRC it should not be turned on by default.

Reply:
In it clearly states 'Have a flag to disable server side rendering that defaults to false' in #58536